src/mount/mount.ceph.c

```diff
@@ -212,6 +212,7 @@ static int parse_options(const char *data, struct ceph_mount_info *cmi)
 
 	do {
 		char * value = NULL;
+		char * new_data = NULL;
```

Comment:
`new_name` seems more appropriate?

Reply:
Actually we could reuse the "data" here. I will remove this.